org.eclipse.jdt.ls.core/src/org/eclipse/jdt/ls/core/internal/corext/template/java/PostfixTemplate.java

```diff
@@ -29,6 +30,7 @@ public enum PostfixTemplate {
 	NULL(PostfixPreferences.NULL_ID, JavaPostfixContextType.ID_ALL, PostfixPreferences.NULL_CONTENT, PostfixPreferences.NULL_DESCRIPTION),
 	NOT(PostfixPreferences.NOT_ID, JavaPostfixContextType.ID_ALL, PostfixPreferences.NOT_CONTENT,
 			PostfixPreferences.NOT_DESCRIPTION),
+  OPT(PostfixPreferences.OPT_ID, JavaPostfixContextType.ID_ALL, PostfixPreferences.OPT_CONTENT, PostfixPreferences.OPT_DESCRIPTION),
```

Comment:
Overall, change looks good. I think the last (minor) thing is just to adjust the indentation on the lines you're introducing. Here and in a few other places the indentation is a bit off.

After that I think we can merge.

Reply:
I converted the spaces to tabs. It should be fixed now.